### ISSUE TITLE
NB2 starting date irrigated/rainfed

### ISSUE THREAD

**Issue by WeirdCxychen** [gicait/PyAEZ#106]
Hi, 
Here is a question about the "maiz_ccd_irr.tif & maiz_ccd_rain.tif", after calculating LGPt0, LGPt5, LGPt10 in NB1, the result could cover the whole study area, but in NB2, the starting date of irrigation and rain-fed GeoTIFF results have some nodata value in the study area. How could that happen? Why could the area have a growing period and not have a start day? I would be grateful if you could provide me with some explanation.

My LPGt and starting date of irrigation results are below.

Regards,
Xinyue Chang

![image](https://github.com/gicait/PyAEZ/assets/118345763/b3a845fa-3481-40c2-96b3-d7d19fe5616a)

![image](https://github.com/gicait/PyAEZ/assets/118345763/400e3edf-5579-4438-b22c-2a1e43525be5)

**Comment by NokuzaSezo97** [comment]
@WeirdCxychen,

Based on your images,  if LGPT10 is less than minimum cycle length of maize, the location won't simulate yield because of insufficient days to support for the entire cycle. Perhaps you could check out a pixel location for crop calendar, yield and LGPT10 to observe.

Best regards,

Swun Wunna Htet

**Comment by NokuzaSezo97** [comment]
@WeirdCxychen,

Yes, any crop calendar results with zero means that one of crop suitability assessments was not met for yield simulation.

Regards,

Swun Wunna Htet

**Comment by WeirdCxychen** [comment]
Thank you so much for your explanation in detail!
